utils.js

```diff
@@ -274,6 +274,72 @@ function toBase64(hexString) {
   return Buffer.from(hexString, "hex").toString("base64");
 }
 
+/**
+ * Return the current timestamp in YYYY-MM-DDTHH:MM:SSZ format.
+ *
+ * @returns {string} ISO formatted timestamp, without milliseconds.
+ */
+export function getTimestamp() {
+  return new Date().toISOString().split(".")[0] + "Z";
+}
+
+/**
+ * Method to determine if a license is a valid SPDX license expression
+ *
+ * @param {string} license License string
+ * @returns {boolean} true if the license is a valid SPDX license expression
+ * @see https://spdx.dev/learn/handling-license-info/
+ **/
+export function isSpdxLicenseExpression(license) {
+  const licenseLoweCase = (license || "").toLowerCase();
+  if (!licenseLoweCase) {
+    return false;
+  }
+
+  if (/[(\s]+/gi.test(licenseLoweCase)) {
+    return true;
+  }
+
+  if (licenseLoweCase.endsWith("+")) {
```

Comment:
This is for the "GPL-2.0+" (or latter) scenario. If this would be part of a more complex case it would be covered by the check for ` ` (space) or `(`.